### PR TITLE
Update internal URLs to no longer show .html

### PR DESCRIPTION
**Pull request by wendi-yu**
Changed all the banner URLs so that clicking to another page doesn't
show a URL with a ".html" at the end, eg
 "waterloorocketry.com/competition.html" would now be
 "waterloorocketry.com/competition".

 Also fixed index.html links to show just "waterloorocketry.com"
 rather than "waterloorocketry.com/index.html".

### CODE REVIEW

**Comment by wendi-yu** [comment]
I'm reasonably confident this won't break anything, but I can't test it on a local host. The ".html" removal is for sure fine but I'm going off a stack overflow post for the "/index.html" fix, which is just replacing "index.html" with "/"

File: contact.html [hunk 1]
```diff
@@ -14,18 +14,18 @@
...
-				<li><a href="rockets.html">ROCKETS</a>
-				<li><a href="team.html">TEAM</a>
-				<li><a href="awards.html">AWARDS</a>					
-				<li><a href="events.html">EVENTS</a>
-				<li><a href="contact.html">CONTACT</a>
+				<li><a href="competition">COMPETITION</a>
+				<li><a href="sponsors">SPONSORS</a>
+				<li><a href="rockets">ROCKETS</a>
+				<li><a href="team">TEAM</a>
+				<li><a href="awards">AWARDS</a>
```
Comment (akmorrison) [inline contact.html]: Trailing tabs at the end of the AWARDS line

File: vidar.html [hunk 2]
```diff
@@ -15,18 +15,18 @@
...
-				<li><a href="rockets.html">ROCKETS</a>
-				<li><a href="team.html">TEAM</a>
-				<li><a href="awards.html">AWARDS</a>	
-				<li><a href="events.html">EVENTS</a>
-				<li><a href="contact.html">CONTACT</a>
+				<li><a href="competition">COMPETITION</a>
+				<li><a href="sponsors">SPONSORS</a>
+				<li><a href="rockets">ROCKETS</a>
+				<li><a href="team">TEAM</a>
+				<li><a href="awards">AWARDS</a>
```
Comment (akmorrison) [inline vidar.html]: Single trailing tab at the end of this line

File: wrt1.html [hunk 3]
```diff
@@ -15,18 +15,18 @@
...
-				<li><a href="rockets.html">ROCKETS</a>
-				<li><a href="team.html">TEAM</a>
-				<li><a href="awards.html">AWARDS</a>					
-				<li><a href="events.html">EVENTS</a>
-				<li><a href="contact.html">CONTACT</a>
+				<li><a href="competition">COMPETITION</a>
+				<li><a href="sponsors">SPONSORS</a>
+				<li><a href="rockets">ROCKETS</a>
+				<li><a href="team">TEAM</a>
+				<li><a href="awards">AWARDS</a>
```
Comment (akmorrison) [inline wrt1.html]: A few trailing tabs here

**Review by akmorrison — COMMENTED** [review]
Very small thing, but a few of these lines have trailing whitespace in them, if you could remove that that would be appreciated. You didn't add the whitespace, but its been there for a long time and we have the chance to remove it without breaking the blame.

If you don't wanna, that's fine

File: contact.html [hunk 4]
```diff
@@ -23,7 +23,7 @@
 				<li><a href="sponsors">SPONSORS</a>
 				<li><a href="rockets">ROCKETS</a>
 				<li><a href="team">TEAM</a>
-				<li><a href="awards">AWARDS</a>					
+				<li><a href="awards">AWARDS</a>
```
Comment (akmorrison) [inline contact.html]: There's still a couple tabs here

**Review by akmorrison — APPROVED** [review]
I'm like 90% sure the link to "/" works, typing `waterloorocketry.com/` into a search field does take you to index.html

Can actually totally confirm it works - I edited source on firefox and the link was still fine

**Comment by wendi-yu** [comment]
oopS thanks for the heads up

**Pull request merged**